content/cif-connector/pom.xml

```diff
@@ -27,7 +27,7 @@
     <packaging>content-package</packaging>
     
     <name>CIF Connector - Content Package</name>
-    <description>Content Package of Connector for CCIF hosted on openwhisk</description>
+    <description>Content Package of Connector for CIF hosted on openwhisk</description>
```

Comment:
Please update the description, this is no longer "hosted on openwhisk".

Reply:
Any proposal for a new description?

Reply:
Just remove "hosted on openwhisk" and it's fine ;-)